Source/Controllers/Other/SPHistoryController.m

```diff
@@ -75,10 +75,6 @@ - (void)dealloc
 {
 	[[NSNotificationCenter defaultCenter] removeObserver:self];
 
-	SPClear(tableContentStates);
-	SPClear(history);
-
-	[super dealloc];
```

Comment:
If we do have a custom dealloc, do we not need to still call the super one?

Reply:
No!